whisper/src/message.rs

```diff
@@ -271,7 +271,7 @@ impl Message {
 		assert!(params.ttl > 0);
 
 		let expiry = {
-			let after_mining = SystemTime::now().checked_sub(Duration::from_millis(params.work))
+			let after_mining = SystemTime::now().checked_add(Duration::from_secs(params.ttl))
```

Comment:
Should probably be `SystemTime::now() + params.work + params.ttl`? And the var name should be changed to reflect meaning of code

Reply:
Yes, agree